src/Prism.tsx

```diff
@@ -135,11 +135,6 @@ export const Prism = forwardRef<PrismHandle, PrismProps>(
 			UNSAFE_vegaSpec,
 		});
 
-		// if the spec has changed, reset the chartView
-		useEffect(() => {
-			chartView.current = undefined;
-		}, [spec]);
-
```

Comment:
This isn't needed and was causing issues where the image couldn't be downloaded after a rerender